src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -264,23 +266,52 @@ def setup_precision_plugin(self) -> None:
                 )
         assert isinstance(self.model, (pl.LightningModule, _LightningPrecisionModuleWrapperBase))
         pl_module = self.model
-        process_group = ProcessGroup()
+
+        new_version_flag = compare_version("colossalai", operator.ge, "0.1.10")
+
```

Comment:
How important is it to support the old code path? Shouldn't we set the min version to 0.1.10?

Reply:
we should definitely support the old code path. the new version is updated for training efficiency, but it is not a quite stable version. we will fix some potential bugs and add more practical functionalities soon. since 0.1.10 is a stable version, we should keep it currently.